piwheels/slave/builder.py

```diff
@@ -211,14 +220,30 @@ def metadata(self):
         """
         if self._metadata is None:
             with zipfile.ZipFile(self.open()) as wheel:
-                filename = (
+                filenames = {
```

Comment:
This is a set containing two values, but it's not obvious that's the case. Each part of the filename structure being on a separate line makes it harder to parse. If each one doesn't fit on a single line I'd probably create two string variables and create a set containing them both (or go with `for filename in {foo, bar}`), although admittedly, naming them would be tricky.